src/components/YourCompany/YourCompany.vue

```diff
@@ -114,29 +206,50 @@
 <script lang="ts">
 import { Component, Emit, Mixins, Watch } from 'vue-property-decorator'
 import { Action, Getter } from 'vuex-class'
-import { ActionBindingIF, BusinessContactIF, GetterIF, IncorporationFilingIF, NameRequestIF } from '@/interfaces'
+import {
+  ActionBindingIF,
+  BusinessContactIF,
+  BusinessSnapshotIF,
+  ConfirmDialogType,
```

Comment:
ConfirmDialogType is an IF?

Reply:
We ref the confirm dialog and define it on the component with this type and i believe it exports from the interfaces.